ui/app/components/flash-toast.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { capitalize } from '@ember/string';
+import Component from '@glimmer/component';
+
+/**
+ * FlashToast components are used to translate flash messages into toast notifications.
+ * Flash object passed should have a `type` and `message` property at minimum.
+ */
+export default class FlashToastComponent extends Component {
+  get color() {
+    switch (this.args.flash.type) {
+      case 'info':
+        return 'highlight';
+      case 'danger':
+        return 'critical';
+      case 'warning':
```

Comment:
is this supposed to return 'warning'?

Reply:
This is a shorthand for `when type is warning or success, return the type`